src/main/java/controller/StudentsOperationsClassic.java

```diff
@@ -0,0 +1,73 @@
+package controller;
+
+import model.Student;
+
+import java.util.*;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/2/2019.
+ */
+public class StudentsOperationsClassic {
```

Comment:
Think how StudentOperations can be organized to get advantages of OOP

Reply:
done - removed logic of static methods. The class was removed as redundant